docs/tools/CLI/cli-setup/cli-reqs.md

```diff
@@ -1,236 +1,139 @@
-
-- **Git and Mercurial:** Mbed CLI supports both Git and Mercurial repositories, and you may need libraries from both sources as you work, so please install both:
-    - [Git](https://git-scm.com/) - version 1.9.5 or greater.
-    - [Mercurial](https://www.mercurial-scm.org/) - version 2.2.2 or greater.
-
-**Optional: Using a virtual environments**
-
-If your main environment has an unsupported Python version, you may want to install Mbed CLI on a virtual environment. Mbed CLI is compatible with [Virtual Python Environment (virtualenv)](https://pypi.python.org/pypi/virtualenv).
-
-## 2. Install a compiler
-
-Download and install one of the following compilers: GCC Arm, Arm Compiler 5, Arm Compiler 6 or IAR. To download the latest toolchains, visit the [supported compilers page](../tools/index.html#compiler-versions).
-
-<span class="notes">**Note:** When installing the Arm Compiler 5 on a 64-bit Linux machine, you may also need to install the i386 architecture package:</span>
-
-    ```
-    $ sudo dpkg --add-architecture i386
-    $ sudo apt-get update
-    $ sudo apt-get install libc6:i386 libncurses5:i386 libstdc++6:i386
-    ```
-
-## 3. Install Mbed CLI
-
-To install Mbed CLI, run `pip install mbed-cli` from your command-line.
+```
+pip install -U mbed-cli
+```
 
-You can verify Mbed CLI installed correctly by running `mbed help`.
+## 3. Install a compiler
 
-## 4. Set up the environment
+Download and install one of the following compilers: [GCC Arm, Arm Compiler 5, Arm Compiler 6 or IAR](../tools/index.html#compiler-versions).
 
-For any installed toolchain, be sure to add the Mbed CLI global configuration:
+**Note:** When installing the Arm Compiler 5 on a 64-bit Linux system, you may also need to install the i386 architecture package. As as example on Ubuntu:
 
 ```
-$ mbed config -G ARM_PATH <path to ARM bin\>
-[mbed] <path to ARM bin\> now set as global ARM_PATH
-
-$ mbed config --list
-[mbed] Global config:
-ARM_PATH=<path to ARM bin\>
-
+$ sudo dpkg --add-architecture i386
+$ sudo apt-get update
+$ sudo apt-get install libc6:i386 libncurses5:i386 libstdc++6:i386
 ```
 
-<span class="notes">**Note:** You can also apply the same configuration to the IAR and GNU toolchains using `IAR_PATH` or `GCC_ARM_PATH`.</span>
-
-# After installation - configuring Mbed CLI
+## 4. Configure the compiler location
 
-Mbed CLI may not work properly without some manual configuration.
+To build project, Mbed CLI needs to know where the compiler is installed. It checks three configuration options in the following order:
 
-## Mandatory: setting PATH variables
+1. Mbed CLI config (local first, then global).
+1. Environment variables.
+1. System path.
 
-Add the following to the system `PATH`:
+<!--the order in the list is the same as the original text, but the content has been changed so that PATH is first rather than last. Why?-->
```

Comment:
I've changed the order because I think installing compilers to the PATH is the most common scenario that people will use. It also doesn't require any additional Mbed CLI commands.

Reply:
Ah I was mistaken, I thought installing it to the PATH was on by default for GCC atleast, turns out this isn't the case. I will switch the order back.